exercises/bowling/bowling_test.go

```diff
@@ -0,0 +1,68 @@
+package bowling
+
+import "testing"
+
+const targetTestVersion = 1
+
+func TestTestVersion(t *testing.T) {
+	if testVersion != targetTestVersion {
+		t.Fatalf("Found testVersion = %v, want %v", testVersion, targetTestVersion)
+	}
+}
+
+func applyPreviousRolls(g *Game, rolls []int) error {
+	for _, pins := range rolls {
+		if err := g.Roll(pins); err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+func TestScore(t *testing.T) {
+	for _, tc := range scoreTestCases {
+		g := NewGame()
+		err := applyPreviousRolls(g, tc.previousRolls)
+		if err != nil {
+			t.Fatalf("%s :\n	unexpected error \"%s\" applying Previous Rolls: %#v before Score()",
+				tc.description, err, tc.previousRolls)
+		}
+		score, err := g.Score()
+		if tc.valid {
+			if err != nil {
+				t.Fatalf("%s : Score() after Previous Rolls: %#v expected %d, got error %s",
+					tc.description, tc.previousRolls, tc.score, err)
```

Comment:
Potential to apply #275 here (ensuring the second return value is of type `error`). I'm not sure it's necessary, but we could either add a `var _ error = err` or call `err.Error()` explicitly? Or should we not bother applying #275 here?

I should document #275 somewhere since everyone keeps forgetting it, maybe it's not really necessary if students by this point already understand that the last return value should be of type `error`?